pennylane/templates/embeddings/angle.py

```diff
@@ -43,6 +43,33 @@ class AngleEmbedding(Operation):
             with :math:`N\leq n`
         wires (Iterable): wires that the template acts on
         rotation (str): type of rotations used
+
+    Example:
+
+        Angle embedding encodes the features as rotaion angles of the specified rotation angle.
+
+        .. code-block:: python
+
+            import pennylane as qml
+
+            dev = qml.device('default.qubit', wires=3)
+
+            @qml.qnode(dev)
+            def circuit(feature_vector):
+                qml.AngleEmbedding(features=feature_vector, wires=range(3), rotation='Z')
+                qml.Hadamard(0)
+                return qml.probs(wires=range(3))
+
+            X = [1,2,3]
+
+        Here, we have also used rotation angles RZ. If not specified, RX is used as default.
```

Comment:
Adding these small references should make the rotations clickable :slightly_smiling_face: 
```suggestion
        Here, we have also used rotation angles :class:`RZ`. If not specified, :class:`RX` is used as default.
```